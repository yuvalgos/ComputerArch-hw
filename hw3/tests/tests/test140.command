./dflow_calc tests/test140.dat tests/test140.in p151 d139 d147 p7 d196 d180 p146 p51 p165 p170 p155 d0 d154 p140 d101 d31 p10 d137 p89 d33 p166 p175 p28 p133 d169 d155 d177 d90 d6 d99 d191 d129 d29 d50 d61