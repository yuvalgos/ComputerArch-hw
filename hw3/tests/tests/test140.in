5 31 28 20
0 11 28 9
3 17 13 0
6 5 14 25
2 7 22 26
2 11 22 24
0 25 22 25
5 13 8 26
0 15 9 2
3 10 6 7
4 21 22 29
8 16 4 3
5 7 7 2
3 6 13 5
7 4 23 25
6 22 7 14
6 10 7 2
7 0 13 11
5 22 7 29
2 14 29 27
3 21 28 2
5 4 16 2
1 20 7 28
7 10 18 15
3 12 13 12
5 25 17 28
5 5 2 25
4 12 29 8
8 20 22 31
1 0 23 16
2 26 22 0
3 15 21 13
7 25 15 0
6 13 19 5
1 30 14 15
6 11 31 16
6 20 21 26
7 10 25 29
3 5 12 14
3 1 15 7
4 7 3 6
3 12 30 24
6 10 30 1
3 8 1 13
4 13 15 19
8 20 23 8
7 31 22 14
2 14 12 1
3 6 4 9
5 21 23 12
4 14 26 2
2 18 6 21
7 20 28 5
2 23 2 3
2 17 6 26
3 6 11 22
4 1 15 26
0 7 3 19
0 16 3 1
5 23 2 25
8 11 31 26
6 21 27 2
1 25 6 22
2 27 4 10
4 31 3 9
0 0 8 29
8 19 16 14
8 20 19 18
4 16 22 14
8 14 20 8
4 28 5 12
3 26 25 2
0 19 23 2
3 1 16 26
1 17 8 21
4 22 11 2
5 1 19 27
4 30 0 2
1 10 14 20
6 14 1 30
7 15 31 19
5 26 26 16
0 18 13 3
1 22 26 0
7 7 23 18
6 25 4 25
2 11 28 20
5 15 24 10
7 22 4 28
4 12 17 21
4 15 19 10
6 27 28 0
7 30 19 26
1 18 31 23
3 10 11 11
3 20 22 24
8 20 31 2
8 14 29 29
5 10 26 11
8 6 31 19
0 29 29 14
2 4 20 2
4 6 17 30
2 1 27 2
2 1 12 5
4 11 12 2
5 10 5 7
0 8 1 23
0 26 4 29
7 20 23 17
4 26 19 25
7 11 24 6
6 0 21 30
2 26 20 22
0 2 25 4
1 3 4 9
6 6 15 26
6 15 23 14
1 5 20 19
4 8 24 8
8 3 25 28
4 4 27 1
2 20 7 5
6 5 15 20
2 7 12 24
5 20 13 16
0 28 21 16
2 11 3 14
0 8 31 6
1 29 11 28
7 0 14 19
3 25 21 29
5 12 6 10
0 31 22 10
7 14 6 16
1 8 3 8
7 28 13 16
6 13 17 7
4 31 16 22
2 7 11 27
8 4 24 8
2 14 19 22
8 20 13 24
5 24 16 16
2 24 28 3
4 19 18 16
5 9 13 31
8 19 1 3
3 4 9 29
8 30 8 4
4 29 26 19
4 24 4 4
7 10 1 22
6 30 12 2
8 11 5 27
5 8 18 9
0 28 13 10
1 5 13 6
7 9 16 3
2 27 1 22
6 23 7 11
8 17 7 16
4 31 0 27
3 17 24 30
4 4 14 7
1 14 4 23
4 28 21 22
3 18 7 1
2 8 16 3
2 13 25 3
1 27 3 21
3 2 3 18
4 20 19 13
1 18 6 4
7 13 11 8
7 26 9 12
0 0 14 17
6 12 30 21
4 15 24 26
8 0 22 2
1 8 30 17
2 14 16 6
8 3 23 10
7 12 25 23
2 22 13 19
1 27 25 21
0 27 0 23
1 31 31 2
3 7 15 3
1 14 28 14
1 12 3 0
3 23 16 11
0 29 24 20
4 12 15 9
4 8 16 12
3 22 10 13
5 1 23 13
8 10 1 13
5 21 25 8
7 5 30 12
