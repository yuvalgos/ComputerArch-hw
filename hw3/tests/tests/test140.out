Reading the opcodes latency info from tests/test140.dat ... Got latency for 9 opcodes
Reading the program file tests/test140.in ... Found 200 instructions
getProgDepth()==227
getDepDepth(151)==113
getInstDeps(139)=={127,91}
getInstDeps(147)=={104,120}
getDepDepth(7)==0
getInstDeps(196)=={191,174}
getInstDeps(180)=={153,163}
getDepDepth(146)==93
getDepDepth(51)==43
getDepDepth(165)==144
getDepDepth(170)==90
getDepDepth(155)==106
getInstDeps(0)=={-1,-1}
getInstDeps(154)=={123,91}
getDepDepth(140)==154
getInstDeps(101)=={96,-1}
getInstDeps(31)=={20,7}
getDepDepth(10)==0
getInstDeps(137)=={74,124}
getDepDepth(89)==60
getInstDeps(33)=={-1,26}
getDepDepth(166)==60
getDepDepth(175)==162
getDepDepth(28)==31
getDepDepth(133)==90
getInstDeps(169)=={131,120}
getInstDeps(155)=={93,146}
getInstDeps(177)=={153,61}
getInstDeps(90)=={72,78}
getInstDeps(6)=={-1,-1}
getInstDeps(99)=={64,72}
getInstDeps(191)=={68,154}
getInstDeps(129)=={127,126}
getInstDeps(29)=={-1,11}
getInstDeps(50)=={30,-1}
getInstDeps(61)=={-1,-1}
